src/qiskit_sphinx_theme/__init__.py

```diff
@@ -67,15 +67,11 @@ def remove_thebe_if_not_needed(
     ]
 
 
-def activate_themes(app: sphinx.application.Sphinx, config: sphinx.config.Config) -> None:
-    if config.html_theme in ["qiskit", "_qiskit-ecosystem"]:
-        # We set a low priority so that our Qiskit CSS file overrides Furo.
-        app.add_css_file("styles/furo.css", 100)
-        app.add_js_file("scripts/qiskit-sphinx-theme.js")
```

Comment:
I deleted this line. We later remove it below, so there is no reason to add it here. But also it doesn't matter if we have it here - inheriting the Furo theme means their own `__init__.py` code will run with this same line.